README.md

```diff
@@ -54,6 +54,9 @@ To further configure your plugin, use the following custom parameters in your `s
 | `integrationTesting`          | Set `true` when running integration tests. This bypasses the validation of the Forwarder ARN and the addition of Datadog Monitor output links. Defaults to `false`. |
 | `logLevel`                    | The log level, set to `DEBUG` for extended logging. |
 | `skipCloudformationOutputs`   | Set to `true` if you want to skip adding Datadog Cloudformation Outputs for your stack. This is useful if you are running into the 200 output limit which can cause the stack creation to fail. |
+| `enableColdStartTracing` | Set to `false` to disable Cold Start Tracing. Used in NodeJS and Python. Defaults to `true`. |
+| `coldStartTraceMinDuration` | Sets the minimum duration for a module load event to be traced via Cold Start Tracing. Number. Defaults to `3` |
```

Comment:
Is this in seconds? If so, should we mention it?

Reply:
Just saw other comments. Resolving.

Reply:
Millis!